### PR TITLE
Add installing using dnf/yum to README

### PR DESCRIPTION
**Pull request by chancez**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,11 +33,29 @@ jq programs are stable and should be considered a bug if it does not match jq be
 
 ### Linux
 
+Static binary from Github Release:
```
Comment (jzelinskie) [inline README.md]: `#### Compiled static binary release`

File: README.md [hunk 2]
```diff
@@ -33,11 +33,29 @@ jq programs are stable and should be considered a bug if it does not match jq be
...
+
 ```sh
 curl -Lo /usr/local/bin/faq https://github.com/jzelinskie/faq/releases/download/0.0.5/faq-linux-amd64
 chmod +x /usr/local/bin/faq
 ```
 
+RPM based install via COPR repositories: https://copr.fedorainfracloud.org/coprs/ecnahc515/faq/ (https://github.com/chancez/faq-rpm)
+
+Fedora:
```
Comment (jzelinskie) [inline README.md]: `#### Fedora`

File: README.md [hunk 3]
```diff
@@ -33,11 +33,29 @@ jq programs are stable and should be considered a bug if it does not match jq be
...
+
+```
+dnf copr enable ecnahc515/faq
+dnf install faq
+```
+
+CentOS:
```
Comment (jzelinskie) [inline README.md]: `#### CentOS`

**Comment by chancez** [comment]
Fixed review items @jzelinskie

**Pull request merged**